tag.hbs

```diff
@@ -16,17 +16,11 @@ into the {body} of the default.hbs template --}}
     <div class="m-hero__content" data-aos="fade-down">
       <h1 class="m-hero-title bigger">{{name}}</h1>
       {{#if description}}
-        <p class="m-hero-description">
-          {{description}}
-        </p>
+        <div class="m-tag__description">{{ description }}</div>
       {{/if}}
-      <div class="m-hero__meta">
-        <ul class="m-hero-stats">
-          <li>
-            {{plural ../pagination.total empty=(t "No posts") singular=(t "1 post") plural=(t "% posts")}} {{t "with this tag"}}
-          </li>
-        </ul>
-      </div>
+      <p class="m-heading__description">
```

Comment:
Please keep it like it is now, there are some scenarios where the class `m-heading__description` doesn't work here.